frontEnd/app/styles/speakers.scss

```diff
@@ -0,0 +1,97 @@
+.speakers{
+  background-color: #D3D3D3;
+}
+
+.speaker-head{
+  font-style: normal;
+  font-size: 3em;
+  padding-top: 0.8em;
+  padding-bottom: 1.4em;
+  font-weight: 500;
+  background-color: #D3D3D3;
+}
+
+.speaker-body{
+  padding: 0;
+  margin: 0;
+}
+
+.speaker-img{
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+}
+
+.speaker-title{
+  font-family: proxima-nova-bold;
+  font-size: 20pt;
+  padding: 0;
+  margin: 0;
+}
+
+.speaker-info{
+  font-family: proxima-nova-regular;
+  font-size: 10pt;
+  padding: 0;
+  margin: 0;
+}
+
+
+.caption-1{
+  width: 100%;
+  color: #fff;
+  bottom: 0rem;
+  position: absolute;
+  background-color: #EC471F;
+  visibility: hidden;
+  opacity: 0.7;
+}
+
+.caption-2{
+  width: 100%;
+  color: #fff;
+  bottom: 0rem;
+  position: absolute;
+  background-color: #EC471F;
+  visibility: hidden;
+  opacity: 0.7;
+}
+
+.caption-3{
+  width: 100%;
+  color: #fff;
+  bottom: 0rem;
+  position: absolute;
+  background-color: #EC471F;
+  visibility: hidden;
+  opacity: 0.7;
+}
+
+.speaker-1{
+  margin: 0;
+}
+
+.speaker-2{
+  margin: 0;
+}
+
+.speaker-3{
+  margin: 0;
+}
+
+.speaker-1:hover .caption-1{
+  visibility: visible;
+}
+
+.speaker-2:hover .caption-2{
+  visibility: visible;
+}
+
+.speaker-3:hover .caption-3{
+  visibility: visible;
+}
```

Comment:
you can use same class name for different html elements. No need to define same property again and again.
Always try not to repeat your code